models.py

```diff
@@ -29,9 +29,19 @@ def delete_genero(nome):
 ##############################################################################
 ############################################################################## Funções - Diretor
 
-def get_diretor(nome):
-    return query("SELECT * FROM diretores")
+def get_diretor(nome_completo):
+    return query(f"SELECT * FROM diretores WHERE nome_completo LIKE %s", (nome_completo,))
+
+
+def insert_diretor(nome_completo):
+    insert('diretores', ('nome_completo',), (nome_completo,))
+
+
+def delete_diretor(nome_completo):
+    delete("diretores", "nome_completo", nome_completo)
+    query("SELECT * FROM diretores")
```

Comment:
se você for usar uma query, não esquecer do return, mas eu removeria esse item aqui